skfp/preprocessing/filters/rule_of_2.py

```diff
@@ -0,0 +1,91 @@
+from typing import Optional
+
+from rdkit.Chem import Mol
+from rdkit.Chem.Crippen import MolLogP
+from rdkit.Chem.Descriptors import MolWt
+from rdkit.Chem.rdMolDescriptors import CalcNumHBA, CalcNumHBD
+
+from skfp.bases.base_filter import BaseFilter
+
+
+class RuleOf2(BaseFilter):
+    """
+    Rule of 2 defined  in [1].
```

Comment:
Avoid citing references in the first line. The first line renders in the docs in summary tables, so it should be particularly short and self-contained.

Here, "Rule of two (Ro2)." would suffice.

Reply:
Also, note that each reference must have an underscore, e.g. `[1]_`, so that it gets properly linked.

Reply:
Done